client/packages/host/src/Admin/LogFileModal.tsx

```diff
@@ -13,8 +13,15 @@ export const LogFileModal = ({
 }) => {
   const t = useTranslation('common');
   const { Modal } = useDialog({ isOpen });
-  const { readLog } = useNativeClient();
-  const [logText, setLogText] = useState<string>('');
+  const { readLog, saveFile } = useNativeClient();
+  const [logText, setLogText] = useState('');
+  const [isSaving, setIsSaving] = useState(false);
+
+  const saveLog = async () => {
+    setIsSaving(true);
+    await saveFile({ content: logText, filename: 'exported_log.txt' });
```

Comment:
planning to update this when the file selection comes in, with the other log viewer issues